scripts/us_eia/cbecs/process.py

```diff
@@ -0,0 +1,863 @@
+# You may obtain a copy of the License at
+#
+#         https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an 'AS IS' BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+'''
+Generate statvars for commercial buildings data set.
+'''
+
+import csv
+import os
+import re
+import sys
+import datetime
+import time
+
+from absl import app
+from absl import flags
+from inspect import getframeinfo, stack
+
+
+# Allows the following module imports to work when running as a script
+# module_dir_ is the path to where this code is running from.
+_MODULE_DIR = os.path.dirname(__file__)
+sys.path.append(os.path.join(_MODULE_DIR))
+sys.path.append(os.path.join(_MODULE_DIR,
+                             '../../../util/'))  # for statvar_dcid_generator
+
+from statvar_dcid_generator import get_statvar_dcid
+from config import PROPERTY_MAP
+from stat_var_group import add_stat_var_groups
+
+FLAGS = flags.FLAGS
+
+flags.DEFINE_list('csv_data_files', [],
+                  'csv files from UNData Energy datasets to process')
+flags.DEFINE_string('output_path', 'tmp_data_dir/un_energy_output',
+                    'Data set name used as file name for mcf and tmcf')
+flags.DEFINE_integer('skip_rows', 0,
+                     'Number of rows to skip at the start of the csv file.')
+flags.DEFINE_integer('header_rows', 1,
+                     'Number of rows to process as header per csv file.')
+flags.DEFINE_integer('num_rows', -1,
+                     'Number of rows to process per file. -1 for all.')
+flags.DEFINE_integer('debug_level', 0, 'Print debug messages at this level or lower.')
+flags.DEFINE_integer('debug_lines', 10, 'Print error logs every N lines')
+flags.DEFINE_boolean('emit_all_properties', False,
+                     'Emit all properties including non-existing ones.')
+flags.DEFINE_boolean('drop_duplicate_svobs', True,
+                     'Drop statvars with duplicate observations.')
+flags.DEFINE_boolean(
+    'schemaless_statvars', True,
+    'Generate schemaless statvars with popType set to the statvar itself.')
+flags.DEFINE_string('svg_root', 'eia/g/Root',
+                    'Generate statvar groups under this root statvar.')
+flags.DEFINE_string('svg_prefix', '',
+                    'Generate statvar groups under this root statvar.')
+
+_DEFAULT_STAT_VAR_PV = {
+    'typeOf': 'dcs:StatisticalVariable',
+    'measurementQualifier': 'Annual',
+    'statType': 'dcs:measuredValue',
+    'observationAbout': 'dcid:country/USA',
+    'populationType': 'CommercialBuilding',
+    'measuredProperty': 'count',
+}
+
+_OBSERVATION_PROPS = [
+    'observationAbout',
+    'observationDate',
+    'value',
+    'variableMeasured',
+    'unit',
+    'Input',
```

Comment:
hm, i don't understand this as part of observation props -- can you add a brief comment?